verification/src/proto/desc/description.proto

```diff
@@ -0,0 +1,69 @@
+// Compile using:
+// $ protoc desc/description.proto -I=desc/ -I=googleapis/  --openapi_out=.
+// while at blockscout-rs/verification/src/proto
+
+syntax = "proto3";
+
+option go_package = "./";
```

Comment:
Smh it doesn't work without it:
https://github.com/evilsocket/opensnitch/issues/373#issuecomment-803663343